hack/kubevirt.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export KUBEVIRT_TAG="${KUBEVIRT_TAG:-main}"
+
+_base_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+_kubectl="${_base_dir}/_kubevirt/cluster-up/kubectl.sh"
+_kubessh="${_base_dir}/_kubevirt/cluster-up/ssh.sh"
+_action=$1
+shift
+
+function kubevirt::install() {
+    if [[ ! -d ${_base_dir}/_kubevirt ]]; then
```

Comment:
Quotes needed?

Reply:
shellcheck is still happy with this commit.

```
$ shellcheck hack/kubevirtci.sh hack/sync.sh hack/kubevirt.sh 

In hack/sync.sh line 17:
source ./hack/common.sh
       ^--------------^ SC1091 (info): Not following: ./hack/common.sh was not specified as input (see shellcheck -x).

For more information:
  https://www.shellcheck.net/wiki/SC1091 -- Not following: ./hack/common.sh w...
```